CHANGELOG.md

```diff
@@ -16,15 +16,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - `StructVariant`
     - `TupleStructVariant`
 - **added:** Add `EnumType::variants_len` ([#94])
+- **added:** Support setting a default value for `OpaqueType`
 
 [#92]: https://github.com/EmbarkStudios/mirror-mirror/pull/92
-[#93]: https://github.com/EmbarkStudios/mirror-mirror/pull/93
 [#94]: https://github.com/EmbarkStudios/mirror-mirror/pull/94
 
 # 0.1.4 (08. February, 2023)
```

Comment:
preexisting: is the release date correct here?

Reply:
It was not. Good catch!